Utility/DynamicJsonParser.cs

```diff
@@ -135,6 +137,55 @@ private static dynamic GetDynamic(PSObject obj, string numericalPrefix = "n", bo
                     propertyName = propertyName.ToLowerInvariant();
                 }
 
+                // Nuspec contains xml metadata as a string (only applicable to VSTS feeds)
+                if (propertyName == "nuspec")
+                {
+                    XmlDocument xmlDoc = new XmlDocument();
+                    string xmlString = (string)psProperty.Value;
+
```

Comment:
so the nuspec property contains xml but sometimes there is something else before the XML that we need to skip, is that right? That seems slightly odd - the xml parser should be able to cope with leading spaces, for example

Reply:
Probably not really a ? but a non-printing character which is shown as ?.